frame/staking/src/lib.rs

```diff
@@ -732,6 +731,143 @@ impl<AccountId, Balance: Default + HasCompact> Default for Exposure<AccountId, B
 	}
 }
 
+impl<AccountId: Clone, Balance: HasCompact + AtLeast32BitUnsigned + Copy>
+	Exposure<AccountId, Balance>
+{
+	/// Splits an `Exposure` into `ExposureOverview` and multiple chunks of `IndividualExposure`
+	/// with each chunk having maximum of `page_size` elements.
+	fn into_pages(
+		self,
+		page_size: u32,
+	) -> (ExposureOverview<Balance>, Vec<ExposurePage<AccountId, Balance>>) {
+		let individual_chunks = self.others.chunks(page_size as usize);
+		let mut exposure_pages: Vec<ExposurePage<AccountId, Balance>> =
+			Vec::with_capacity(individual_chunks.len());
+
+		for chunk in individual_chunks {
```

Comment:
If you use `into_iter()` here, you will be owning each chunk and therefore able to assign it to `ExposurePage.others` without needing to copy anything (I hope).